golang/explode.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"errors"
+	"strings"
+)
+
+func Explode(separator string, str string, limit ...int) []string {
+	if separator == "" {
+		return []string{errors.New("Separator cannot be empty").Error()}
+	}
+
+	var result []string
+	result = strings.Split(str, separator)
```

Comment:
```suggestion
	var result = strings.Split(str, separator)
```

Nggak perlu new line atau reassignment.